make_level.py

```diff
@@ -0,0 +1,217 @@
+import sys
+import os
+from shutil import copyfile
+
+cwd = os.getcwd()
+
+
+def write_init(skill_name, skill_path, level_name):
+    # If skills/skill/__init__.py doesn't exist, create a basic version
+    if not os.path.exists(os.path.join(skill_path, "__init__.py")):
+        with open(os.path.join(skill_path, "__init__.py"), 'w+') as fp:
+            level_setup = "\n".join([
+                "import pkg_resources",
+                "",
+                "from gitgud.skills.util import BasicLevel",
+                "from gitgud.skills.util import Skill",
+                "",
+                "skill = Skill(",
+                "    '{}',".format(skill_name),
+                "    [",
+                "    ]",
+                ")",
+                ""
+                ])
+            fp.write(level_setup)
+
+        # Read file
+        with open(os.path.join("gitgud", "skills", "__init__.py"), 'r') as fp:
+            filedata = fp.read()
+
+        # Add import statement
+        replace = "\n".join([
+            "from gitgud.skills.{0} import skill as {0}_skill".format(skill_name),
+            "",
+            "from gitgud.skills.u"
+        ])
+        filedata = filedata.replace("\nfrom gitgud.skills.u", replace)
+
+        # Add to input array of Skill
+        replace = ",\n    {}_skill\n]".format(skill_name)
+        filedata = filedata.replace("\n]", replace)
+
+        # Write to file
+        filepath = os.path.join("gitgud", "skills", "__init__.py")
+        with open(filepath, 'w') as fp:
+            fp.write(filedata)
+            print("Registered skill \"{}\" in {}".format(skill_name, filepath))
+
+        write_init(skill_name, skill_path, level_name)
```

Comment:
Since there's multiple `__init__.py`s you could be writing do, can you find a way to signal that this one is in the new skill's directory. That can either be by renaming the function or just a comment

Reply:
Hmmm, the write_init function writes both skill/__init__.py and skill/<new_skill>/__init__.py. The script adds to the former if the file doesn't exist, then the latter is written to recursively. If the file exists, then only the latter written to.

Reply:
I can keep the function name and then write a comment above it explaining this, or I can change the name; let me know.